code/modules/antagonists/traitor/IAA/internal_affairs.dm

```diff
@@ -30,6 +30,15 @@
 
 /datum/antagonist/traitor/internal_affairs/on_gain()
 	START_PROCESSING(SSprocessing, src)
+	
+	//Gives Cyanide dental implant
+	var/obj/item/reagent_containers/pill/cyanide/cyan = new
+	owner.current.transferItemToLoc(cyan, target, TRUE)
```

Comment:
this `target ` isn't defined as far as I can tell

Reply:
Did you mean ``owner``?

Reply:
I did mean ``owner.``